chain-impl-mockchain/src/block/mod.rs

```diff
@@ -222,6 +222,10 @@ mod test {
         fn block_serialization_bijection(b: Block) -> TestResult {
             property::testing::serialization_bijection(b)
         }
+
+        fn block_is_consient(block: Block) -> TestResult {
```

Comment:
typo

Reply:
I couldn't find a use for this function, is there something missing maybe ?

Reply:
Removing